db/src/bootstrap.rs

```diff
@@ -198,9 +197,8 @@ fn symbolic_schema_to_triples(ident_map: &IdentMap, symbolic_schema: &Value) ->
                             // schema directly as Rust data.
                             let typed_value = match TypedValue::from_edn_value(value) {
                                 Some(TypedValue::Keyword(ref s)) => {
-                                    let entid = to_namespaced_keyword(s)
-                                        .and_then(|ident| ident_map.get(&ident))
-                                        .ok_or(ErrorKind::UnrecognizedIdent(s.clone()))?;
+                                    let entid = ident_map.get(s)
```

Comment:
There might be a `require` helper for `IdentityMap` lurking in here, like in `db/src/schema.rs`.  That'll give you one place for all these TODO references.